ocrd-pagetopdf

```diff
@@ -103,6 +103,7 @@ main () {
         if [ -n "$img_file_grp" ]; then
             img_file=$(ocrd workspace find \
                             -g $pageid -G $img_file_grp \
+                            -m //image/.* \
```

Comment:
asterisk probably best quoted or escaped

Reply:
```suggestion
                            -m '//image/.*' \
```

Reply:
Like that?